docs/examples/ex_construct.py

```diff
@@ -0,0 +1,21 @@
+from pydantic import BaseModel, ValidationError
+
+class Model(BaseModel):
+    a: int
+
+valid_data = dict(a=5)
+invalid_data = dict(a="dog")
+
+# Validate the model at instantiation
+m = Model(**valid_data)
+print(m)
+try:
+    Model(**invalid_data)
+except ValidationError as e:
+    print(e)
```

Comment:
I don't think there's a need for this section, it's already covered in other examples.

Best just to give an example of the correct usage of `construct()`, then have a prominent note (perhaps a `!!! warning`) about misuse of of the method.